plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/meta/CubridMetaModel.java

```diff
@@ -53,100 +65,362 @@ public String getTableOrViewName(GenericTableBase base)
         return null;
     }
 
+    @Nullable
     @Override
-    public List<GenericSchema> loadSchemas(JDBCSession session, GenericDataSource dataSource, GenericCatalog catalog)
-            throws DBException
-    {
+    public List<GenericSchema> loadSchemas(@NotNull JDBCSession session, @NotNull GenericDataSource dataSource, @Nullable GenericCatalog catalog)
+            throws DBException {
         List<GenericSchema> users = new ArrayList<>();
         try {
             final JDBCPreparedStatement dbStat = session.prepareStatement("select * from db_user");
             dbStat.executeStatement();
             JDBCResultSet dbResult = dbStat.getResultSet();
-
-            while (dbResult.next()) {
-                String name = JDBCUtils.safeGetStringTrimmed(dbResult, "name");
-                String description = JDBCUtils.safeGetStringTrimmed(dbResult, "comment");
-                CubridUser user = new CubridUser(dataSource, name, description);
-                users.add(user);
+            try {
+                while (dbResult.next()) {
+                    String name = JDBCUtils.safeGetStringTrimmed(dbResult, "name");
+                    String description = JDBCUtils.safeGetStringTrimmed(dbResult, "comment");
+                    CubridUser user = new CubridUser(dataSource, name, description);
+                    users.add(user);
+	            }
+            } finally {
+                dbStat.close();

```

Comment:
Try-catch with resources.

Reply:
Use try (JDBCPreparedStatement dbStat = session.prepareStatement("select * from db_user")).
No need to use explicit dbStat.close().

Reply:
I have fixed this.